libkbfs/mdserver_local_shared.go

```diff
@@ -61,6 +63,51 @@ func isWriterOrValidRekey(ctx context.Context, codec Codec, kbpki KBPKI,
 		ctx, codec, kbpki, mergedMasterHead, true, newMd)
 }
 
+type mdServerLocalTruncateLockManager struct {
+	// TLF ID -> device KID.
+	locksDb map[TlfID]keybase1.KID
```

Comment:
No mutex for the map?  If you're relying on the higher level mutex, a comment would probably be good.